frontend/javascripts/oxalis/view/version_entry_group.tsx

```diff
@@ -18,6 +18,39 @@ type State = {
   expanded: boolean;
 };
 
+function GroupHeader({
+  toggleExpand,
+  expanded,
+  batches,
+}: {
+  toggleExpand: () => void;
+  expanded: boolean;
+  batches: APIUpdateActionBatch[];
+}) {
+  const lastTimestamp = _.max(batches[0].value.map((action) => action.value.actionTimestamp)) || 0;
+  const lastVersion = _.last(batches)?.version || 0;
+  return (
+    <List.Item
+      style={{
+        cursor: "pointer",
+      }}
+      className="version-entry version-entry-header"
+      onClick={toggleExpand}
+    >
+      <List.Item.Meta
+        title={
+          <React.Fragment>
+            {lastVersion} to {batches[0].version} (
+            <FormattedDate timestamp={lastTimestamp} format="HH:mm" />)
+          </React.Fragment>
+        }
+        avatar={
+          <Avatar size="small" icon={expanded ? <CaretDownOutlined /> : <CaretRightOutlined />} />
+        }
+      />
+    </List.Item>
+  );
+}
```

Comment:
I moved `GroupHeader` out of `VersionEntryGroup` because of some typescript warning. While doing so I got rid of 2 ignored typescript warnings.